builds/jazz-build-module/sls-app/whitelist.yml

```diff
@@ -462,3 +462,14 @@ plugins:
   - serverless-mocha-plugin
   - serverless-kms-secrets
     
+arnTemplate:
```

Comment:
Need to check what other resource types we need to add to assets and add here.

Reply:
Please check the dynamodb stream, cloudwatch event, kinesis stream, and sqs also